### DISCUSSION TITLE
Virustotal

### DISCUSSION THREAD

**Discussion by NoctureaTeam** [FabricMC/fabric-api#2845] [FAQ] [answered]
Hello the link to install fabricMc is infected by a virus https://fabricmc.net/use/installer/

https://www.virustotal.com/gui/file/c577507a9388b5b4520a8f14474062293e4cd26dfc9421c5acc49c992011ffa6

![Capture d'écran_20230113_162630](https://user-images.githubusercontent.com/120424755/212362164-5f4f1c45-bcfd-410e-a70e-76122d49fcbf.png)

**Comment by modmuss50** [comment, marked as answer]
Hi, this is a false positive from a few unknown AV providers. Well known/commonly trusted AVs do not report it as a virus.

If you are unsure you can use the universal .jar download or build it from sourcecode.

Due to a range of reasons we are unable to sign the Windows biniary this is the main reason we sometimes get flagged.
